### PR TITLE
Update README with more details.

### PR DESCRIPTION
**Pull request by finnlewis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,221 @@
...
+```bash
+cd MY_PROJECT
+lando start
+```
+
+Once lando has finished building, use lando to run composer install and the site installer.
+
+```bash
+lando composer install
+lando drush si localgov -y
```
Comment (msayoung) [inline README.md]: localgov_microsites

File: README.md [hunk 2]
```diff
@@ -1,18 +1,221 @@
...
+If you want to be sure you are getting the latest commits when developing,
+clearing composer cache, deleting the folders and re-running composer update
+seems to be a solid approach:
+
+```bash
+
+rm -rf web/profiles/contrib/ web/modules/contrib/ web/themes/contrib/;
+composer clear-cache; composer update --with-dependencies --no-cache;
+lando start;
+lando drush si localgov -y;
```
Comment (msayoung) [inline README.md]: localgov_microsites

File: README.md [hunk 3]
```diff
@@ -1,18 +1,221 @@
...
+
+```
+
+If you run into [memory limit errors](https://getcomposer.org/doc/articles/troubleshooting.md#memory-limit-errors)
+when running Composer commands, prefix the commands with `COMPOSER_MEMORY_LIMIT=-1`.
+For example, to install the project run:
+
+```bash
+COMPOSER_MEMORY_LIMIT=-1 composer create-project localgovdrupal/localgov-project MY_PROJECT
```
Comment (msayoung) [inline README.md]: localgov-microsites-project

File: README.md [hunk 4]
```diff
@@ -1,18 +1,221 @@
...
-This profile creates a single node of demo / default content using the https://www.drupal.org/project/default_content module.
+* [PHPUnit documentation](https://phpunit.readthedocs.io/en/7.5/)
+* [Drupal 8 PHPUnit documentation](https://www.drupal.org/docs/8/testing/phpunit-in-drupal-8)
+* [Drupal 8 testing documentation](https://www.drupal.org/docs/8/testing)
+* [Workshop: Automated Testing and Test Driven Development in Drupal 8](https://github.com/opdavies/workshop-drupal-automated-testing)
 
-This node includes layout paragraphs and paragraphs to demonstrate some of the components for a new microsite. When a new microsite is created, it attempts to clone this node into the new microsite. 
+## Developer notes
 
 As developers, we often want to update this default content, using drush.
```
Comment (msayoung) [inline README.md]: i think this line can be deleted now

**Review by msayoung — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (finnlewis) [reply README.md]: thanks, done

File: README.md (hunk 2 above)
Comment (finnlewis) [reply README.md]: thanks, done

File: README.md (hunk 4 above)
Comment (finnlewis) [reply README.md]: Renamed the title to "Default content"

File: README.md [hunk 5]
```diff
@@ -1,20 +1,223 @@
...
+```
+
+### Testing resources
+
+* [PHPUnit documentation](https://phpunit.readthedocs.io/en/7.5/)
+* [Drupal 8 PHPUnit documentation](https://www.drupal.org/docs/8/testing/phpunit-in-drupal-8)
+* [Drupal 8 testing documentation](https://www.drupal.org/docs/8/testing)
+* [Workshop: Automated Testing and Test Driven Development in Drupal 8](https://github.com/opdavies/workshop-drupal-automated-testing)
+
+## Default content
```
Comment (msayoung) [inline README.md]: actually I think all this wants to move up in to the Developer notes for default content section above.

**Review by msayoung — CHANGES_REQUESTED** [review]
see one more comment, sorry

File: README.md (hunk 5 above)
Comment (finnlewis) [reply README.md]: any chance you could do that please?

**Review by msayoung — APPROVED** [review]

**Pull request merged**